source/agora/test/Base.d

```diff
@@ -570,23 +577,15 @@ public enum NetworkTopology
 
     Params:
         APIManager = Type of API manager to instantiate
-        topology = Network topology to adopt
-        nodes    = Number of nodes to instantiated
-        configure_network = whether to set up the peers in the config
-        retry_delay = the delay between request retries (in msecs)
-        max_retries = max retries before a request is considered failed
-        timeout = request timeout (in msecs)
-        max_failed_requests = max failed requests before a node is banned
+        test_conf = the test configuration
 
     Returns:
         The set of public key added to the node
 
 *******************************************************************************/
 
-public APIManager makeTestNetwork (APIManager : TestAPIManager = TestAPIManager)
-    (NetworkTopology topology, size_t nodes, bool configure_network = true,
-        long retry_delay = 100, size_t max_retries = 20, long timeout = 500,
-        size_t max_failed_requests = 100)
+public APIManager makeTestNetwork (APIManager : TestAPIManager = TestAPIManager)(
+    in TestConf test_conf = TestConf.init)
```

Comment:
Actually, I changed my mind. I don't want a default value here. I just got bitten by it lol.